Doc/whatsnew/3.13.rst

```diff
@@ -521,6 +521,15 @@ io
   built on debug mode <debug-build>`.
   (Contributed by Victor Stinner in :gh:`62948`.)
 
+inspect
+-------
+
+* Add *bound_arg* parameter to :func:`inspect.Signature.from_callable`
+  and :func:`inspect.signature`: keep the ``self`` parameter
+  in the method signature if *bound_arg* is True.
+  :pypi:`inspect313` package has a backport of this feature.
```

Comment:
Maybe make the backport mention a seealso note in the module docs rather than only putting it here?